indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -277,6 +282,8 @@
             :instrument-sn.lowercase  (map str/lower-case instrument-short-names)
             :sensor-sn sensor-short-names
             :sensor-sn.lowercase  (map str/lower-case sensor-short-names)
+            :authors authors
+            :authors.lowercase (map str/lower-case authors)
```

Comment:
authors can be an empty list, but not nil. The keep functions return an empty list if there is nothing there.